### DISCUSSION TITLE
How to know if a camera is active

### DISCUSSION THREAD

**Discussion by ismaelisuani** [bluenviron/mediamtx#3636] [Questions]
### Question

I am consulting the list of cameras via the API http://localhost:9997/v3/config/paths/list.

Is there any parameter that tells me if the camera is active or not, without pinging the camera's IP, but directly to RTSP?

I have read that the status of the camera can be consulted via RTSP GET_PARAMETER. Is there a possibility of this being displayed via the API?

Thank you very much!

**Comment by aler9** [comment]
Hello, the `/v3/config/paths/list` endpoint does not provide the list of cameras, but the list of camera configurations.

The endpoint that provides the list of cameras is `/v3/paths/list` and has a `ready` field that shows whether a camera is active or not.
